index.html

```diff
@@ -1,551 +1,549 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Shafiu | Portfolio</title>
-    <link rel="stylesheet" href="./index.css">
-</head>
-<body>
+    <link rel="stylesheet" href="./index.css" />
+  </head>
+  <body>
     <div>
-        <header>
-            <nav class="navbar">
-                <div class="nav__logo">
-                    <a href="#">SHAFIU</a> 
+      <header>
+        <nav class="navbar">
+          <a href="#" class="navbar__branding">Shafiu</a>
+          <ul class="nav__menu">
+            <li class="nav__item">
+              <a href="#" class="nav__link nav__mobile-link">Portfolio</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link nav__mobile-link">About</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link nav__mobile-link">Contact</a>
```

Comment:
Just as the previous reviewer [requested](https://github.com/shafiuyushawu/portfolio-/pull/7#discussion_r1091298608) please make sure when the user clicks (or taps) any of the mobile menu options, the correct section of the page is displayed, please add the path to the links above.